### PR TITLE
Notes app

### PR DESCRIPTION
**Pull request by Ayush-Dahala**
# Zero To Mastery - Webblocks 2022 📦📦

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
- In this project you can take your daily notes and delete them as you wanted and search them based on keyword

## Breaking changes
-none

## Screenshots
image

### CODE REVIEW

File: src/blocks/Ayush-Dahala/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+console.log("Welcome to my notes")
```
Comment (sophiabrandt) [inline src/blocks/Ayush-Dahala/index.js:1]: Please remove `console.log()` from production code.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please remove `console.log`.

**Comment by Ayush-Dahala** [comment]
Updated I removed all the console.log

**Comment by Ayush-Dahala** [comment]
> Please remove `console.log`.

yes I removed all the console.log

**Review by MattCSmith — APPROVED** [review]
Looks good to me 👍

**Pull request merged**